Puppetfile.stable

```diff
@@ -108,7 +108,7 @@ mod 'herculesteam-augeasproviders_sysctl',
 
 mod 'onyxpoint-gpasswd',
   :git => 'https://github.com/simp/puppet-gpasswd',
-  :ref => '04431f902da8b20527e1cb5eb98ec6f4fefd6429'
+  :ref => '1.0.3'
```

Comment:
04431 was 1.0.1 and we just updated to 1.0.3 , why is this marked stable?

Reply:
@nick-markowski Because people keep insisting on updating stable even though it's supposed to only be updated at release time.